### ISSUE TITLE
Menu alias does not generate active class

### ISSUE THREAD

**Issue by jmshey90** [gantry/gantry5#3011]
In Joomla, if you create a menu alias, it doesn't generate the active class on the menu item when you land on it.

**Comment by mahagr** [comment]
Can you verify that Joomla default template does this? Also in which version of CMS are you using?

**Comment by hjelperne** [comment]
Yes - I can confirm the issue in v5.5.15 / g5_hydrogen and Joomla 4.3.0 Stable.
Menu items in Main Menu that are aliases do not receive the active class on selected items
